code/modules/tgui/modules/ghost_crew_manifest.dm

```diff
@@ -0,0 +1,17 @@
+GLOBAL_DATUM_INIT(ghost_crew_manifest, /datum/ui_module/ghost_crew_manifest, new)
+
+/datum/ui_module/ghost_crew_manifest
+	name = "Crew Manifest"
+
+/datum/ui_module/ghost_crew_manifest/ui_interact(user, ui_key = "GhostCrewManifest", datum/tgui/ui = null, datum/tgui/master_ui = null, state)
+	ui = SStgui.try_update_ui(user, src, ui_key, ui)
+	if(!ui)
+		ui = new(user, src, ui_key, "GhostCrewManifest", name, 588, 510, master_ui, state)
+		ui.set_autoupdate(FALSE)
+		ui.open()
+
+/datum/ui_module/ghost_crew_manifest/ui_data(user)
+	var/list/data = list()
+	GLOB.data_core.get_manifest_json()
```

Comment:
Though, I guess, theres no harm in showing whos SSD and who isnt ingame. You can manually look at them.

Reply:
Yeah, and SSD fills this weird space between IC and OOC anyway. I can't really imagine a scenario where it can be used to metagame to any substantial degree.